src/platform/forms-system/src/js/components/Attestation.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import FormSignature from './FormSignature';
+import PropTypes from 'prop-types';
+
+/**
+ * Wraps the FormSignature in a gray box and handles the aria-labelledby
+ * attribute by default.
+ *
+ * Example usage in formConfig:
+ * presubmitInfo: {
+ *   CustomComponent: signatureProps => (
+ *     <Attestation heading="Statement of truth">
+ *       <p>I solemnly swear I am up to no good.</p>
+ *     </Attestation>
+ *   )
+ * }
+ */
+export const Attestation = ({ children, heading, ...props }) => {
+  return (
+    <div className="box vads-u-background-color--gray-lightest vads-u-padding-bottom--6 vads-u-padding-x--3 vads-u-padding-top--1px vads-u-margin-bottom--7">
+      {typeof heading === 'string' ? <h2>{heading}</h2> : heading}
+      <div id="attestation-content">{children}</div>
+      <FormSignature
+        {...props}
+        ariaLabelledBy={`${props.ariaLabelledBy ||
+          ''} attestation-content`.trim()}
```

Comment:
I'd like to change the hard-coded ID "attestation-content" to a prop like `descriptionId` so we can pass it into the FormSignature component—and later Checkbox—without devs having to copy and paste the string more than once. I'd also like to change the prop name to `ariaDescribedBy`.

```javascript
<div id={descriptionId}>{children}</div>
<FormSignature {...props} ariaDescribedBy={descriptionId} />
```